include/tvm/te/schedule.h

```diff
@@ -466,9 +498,27 @@ class StageNode : public Object {
    *  while origin_op remains fixed.
    */
   Operation origin_op;
-  /*! \brief All the nodes in the iter var */
+  /*! \brief All the nodes in the iter var
+   *
+   * Each element of all_iter_vars represents an iteration variable
```

Comment:
Thank you!  I wasn't sure if this should be a separate PR, since it doesn't impact functionality, but it took a bit of code diving to figure out the distinction, and so I wanted to make sure to document it to save time later.  (Including for future-Eric, who is liable to forget these subtleties in a month.)